cmd/security-secretstore-setup/Dockerfile

```diff
@@ -36,7 +36,7 @@ RUN make cmd/security-file-token-provider/security-file-token-provider \
 
 FROM alpine:3.12
 
-RUN apk add --update --no-cache ca-certificates dumb-init curl
+RUN apk add --update --no-cache ca-certificates dumb-init curl su-exec
```

Comment:
Do we still need `su-exe`?

Reply:
Already removed it and checked it in...this is "outdated"...